cpp/src/barretenberg/honk/CMakeLists.txt

```diff
@@ -1,5 +1,4 @@
-# TODO(Cody): Remove plonk dependency
```

Comment:
I had already observed with Mara that this was easy to do, and doing so here let me work on this PR successively.